import logging

logger = logging.getLogger(__name__)

def log_postgresql_exception(err):

    pass